.env.example

```diff
@@ -2,6 +2,10 @@ APP_ENV=local
 APP_DEBUG=true
 APP_KEY=SomeRandomString
 
+NORTHSTAR_URL=http://northstar.app:8000
+NORTHSTAR_API_KEY=abc4324
+NORTHSTAR_ID_FIELD=votingapp_id
```

Comment:
does this actually work, since we aren't seeding the NS database with this value?